doc/source/code_ref/coordinates.rst

```diff
@@ -0,0 +1,196 @@
+coordinate frames in to the :ref:`Astropy coordinates system <astropy:astropy-coordinates>`.
+
+.. warning::
+
+   The accuracy of the transformations in this module have not been rigorously
+   verified. They have been compared to `sunpy.wcs` and match to numerical
+   precision. Independent verification will be added at a later date.
+
+
+Getting Started
+---------------
+
+The easiest interface to the coordinates module is through the `~astropy.coordinates.SkyCoord` class::
+
+  >>> import astropy.units as u
+  >>> from astropy.coordinates import SkyCoord
+  >>> import sunpy.coordinates
+  >>> c = SkyCoord(-100*u.arcsec, 500*u.arcsec, frame='helioprojective')
+  >>> c = SkyCoord(90*u.deg, -76*u.deg, frame='helioprojectiveradial')
+  >>> c = SkyCoord(x=-72241.0*u.km, y=361206.1*u.km, z=589951.4*u.km, frame='heliocentric')
+  >>> c = SkyCoord(70*u.deg, -30*u.deg, frame='heliographicstonyhurst')
+  >>> c
+  <SkyCoord (HelioGraphicStonyhurst: dateobs=None): (lon, lat, rad) in (deg, deg, km)
+      (70.0, -30.0, 695508.0)>
+
+
+SunPy implements supports for the following solar physics coordinate systems:
+
+* Helioprojective (Cartesian) `~sunpy.coordinates.frames.HelioProjective`
+* Helioprojective (Radial) `~sunpy.coordinates.frames.HelioProjectiveRadial`
+* Heliocentric `~sunpy.coordinates.frames.HelioCentric`
+* Heliographic Stonyhurst `~sunpy.coordinates.frames.HelioGraphicStonyhurst`
+* Heliographic Carrington `~sunpy.coordinates.frames.HelioGraphicCarrington`
+
+for a complete description of these frames, see `sunpy.coordinates.frames`.
+
+
+`~astropy.coordinates.SkyCoord` and all other `~astropy.coordinates` objects
+also support array coordinates. These work the same as single-value coordinates,
+but they store multiple coordinates in a single object. When you're going to
+apply the same operation to many different coordinates, this is a better choice
+than a list of `~astropy.coordinates.SkyCoord` objects, because it will be
+*much* faster than applying the operation to each
+`~astropy.coordinates.SkyCoord` in a for loop.
+::
+
+   >>> c = SkyCoord([-500, 400]*u.arcsec, [100, 200]*u.arcsec, frame='helioprojective')
+   >>> c
+   <SkyCoord (HelioProjective: D0=149597870.7 km, dateobs=None, L0=0.0 deg, B0=0.0 deg, RSun=695508.0 km): (Tx, Ty) in arcsec
+       [(-500.0, 100.0), (400.0, 200.0)]>
+   >>> c[0]
+   <SkyCoord (HelioProjective: D0=149597870.7 km, dateobs=None, L0=0.0 deg, B0=0.0 deg, RSun=695508.0 km): (Tx, Ty) in arcsec
+       (-500.0, 100.0)>
+
+
+Accessing Coordinates
+^^^^^^^^^^^^^^^^^^^^^
+
+Individual coordinates can be accessed via attributes on the SkyCoord object,
+the names of the components of the coordinates for each frame differ. For a full
+description of all the properties of the frames see `sunpy.coordinates.frames`.
+
+``HelioProjective``::
+
+  >>> c = SkyCoord(-500*u.arcsec, 100*u.arcsec, frame='helioprojective')
+  >>> c.Tx
+  <Longitude180 -500.0 arcsec>
+  >>> c.Ty
+  <Latitude 100.0 arcsec>
```

Comment:
Worth adding a sentence to say what ``Tx`` and ``Ty`` are? Also, why use uppercase letters in the attribute names?